common/locales/en/settings.json

```diff
@@ -140,5 +140,8 @@
     "gemCapExtra": "Gem Cap Extra:",
     "mysticHourglasses": "Mystic Hourglasses:",
     "paypal": "PayPal",
-    "amazonPayments": "Amazon Payments"
+    "amazonPayments": "Amazon Payments",
+    "timezone": "Timezone",
+    "timezoneUTC": "Your timezone is: <b><%= utc %></b>",
```

Comment:
I feel like this should be either `"Your timezone: <b><%= utc %></b>"` or `"Your timezone is <b><%= utc %></b>"`

@lemoness Wording thoughts?